website_typed_text/static/src/js/frontend.js

```diff
@@ -0,0 +1,66 @@
+/* global Typed*/
+odoo.define("website_typed_text.frontend", function(require) {
+    "use strict";
+
+    var snippetAnimation = require("website.content.snippets.animation");
+
+    var AnimatedTextMixin = {
+        _fetchSettingsFromElement: function($el) {
+            var data = $el.data();
+            var settings = {
+                strings: [],
+                typeSpeed: data.typeSpeed,
+                loop: data.loop === 1,
+                backSpeed: data.backSpeed,
+                backDelay: data.backDelay,
+                cursorChar: data.cursorChar,
+                showCursor: data.cursorChar.length > 0,
+                smartBackspace: true,
+            };
+            _.each(data, function(value, key) {
+                if (_.str.startsWith(key, "strings-") && value) {
+                    settings.strings.push(value);
+                }
+            });
+            return settings;
+        },
+    };
+
+    snippetAnimation.registry.snippet = snippetAnimation.Animation.extend(
+        AnimatedTextMixin,
+        {
+            selector: ".typed_text",
+            disabledInEditableMode: true,
+            start: function() {
+                var res = this._super.apply(this, arguments);
+
+                var data = this.$target.data();
+                var strings = [];
+                _.each(data, function(value, key) {
+                    if (_.str.startsWith(key, "strings-")) {
+                        strings.push(value);
+                    }
+                });
+                this.$target.attr("data-original", this.$target.text());
+                this.$target.attr("data-css-height", this.$target[0].style.height);
+                this.$target.height(this.$target.height());
```

Comment:
Why need this?

Reply:
Otherwise the content on the website will move up and down; very annoying :)